doc/Requirements Specification/Parts/Application.lyx

```diff
@@ -272,7 +272,7 @@ gls{PCM}
 
 \end_inset
 
-), which can hence be fixed in time.
+), which can then be fixed early on.
```

Comment:
I think that this part of the sentence can be deleted, so the sentence would be:
>This leads to earlier detection of arising problems (like implementation errors or
unrealistic modelling in the PCM).

I find it redundant to say that the problems can be fixed (as it is obvious) in time and the usage of "early" two times seems to be inelegent.